modules/FormModel.js

```diff
@@ -113,86 +115,240 @@
     },
 
     /**
-     * Add a model that this form model should track against
-     * @method addModel
-     * @param modelConfig {Object} the Object model configuration you are tracking.
-     *   @param modelConfig.model {Backbone.Model} the object model
-     *   @param [modelConfig.fields] {Array} an array of strings where each String value corresponds to an attribute in the model.
-     *     Leave empty if you want to listen to all the fields.
-     * @param [copy=false] {Boolean} set to true if you want to make an initial pull from the object model upon adding.
+     * @method getMapping
+     * @param alias {String} the alias of the mapping - either a model mapping or a computed mapping
+     * @return the mapping config for that alias
      */
-    addModel: function(modelConfig, copy) {
-      this.__modelConfigs.push(modelConfig);
-      if (copy) {
-        this.__copyFields(modelConfig.fields, this, modelConfig.model);
-        this.__updateCache(modelConfig.model);
+    getMapping: function(alias) {
+      return this.__currentMappings[alias];
+    },
+
+    /**
+     * @method getMappings
+     * @return all the current mapping configs
+     */
+    getMappings: function() {
+      return this.__currentMappings;
+    },
+
+    /**
+     * Define or redefine how the form model pull/pushes or otherwise tracks properties between an object model(s).
+     * Examples:
+     * this.setMapping('modelAlias', true, optional model instance);
```

Comment:
I forget what we said in the issue, did we want to support a shortened 'no-alias' setMapping()? Probably not, but that's the only thing that I could see that would be simpler for the common case.

Maybe something better, if you don't have a mapping for a model we push/pull everything?  So the simplest config for setting all is model.setTrackedModel(modelInstance) (i.e. no setMapping(s)())?  Or would that be confusing and unintuitive?